vmd-client/README.md

```diff
@@ -1 +1 @@
-Placeholder
\ No newline at end of file
+UI tool to control app VMs
```

Comment:
The app designed login model should be described here. The current LoginView uses password-model but that is not used for client authentication over the REST API. If the password model prototype concept is only to demonstrate login to the application to access privileged control of VMs, how are the accounts stored in the VM hosting the app?

Reply:
The app login model has not been designed. Where can I find  the requirements for this? Was it discussed during my sick leave?

Reply:
This is the access control algorithm I am going to implement on my side. 
The users must authorize themselves to change settings by providing username and password or by inquiring the one-time pin code. The last entered username(admin/root by default) is saved by the app, if there is no username entered, the user will be asked to enter it or register themselves or get temporary access with pin code sent on entered phone number.
It is just a draft, everything is discussible.